template-manager-core/src/main/java/com/sequenceiq/cloudbreak/template/TemplateModelContextBuilder.java

```diff
@@ -65,11 +64,10 @@ public TemplateModelContextBuilder withGeneralClusterConfigs(GeneralClusterConfi
         return this;
     }
 
-    public TemplateModelContextBuilder withRdsConfigs(Iterable<RdsConfigWithoutCluster> rdsConfigs) {
-        for (RdsConfigWithoutCluster rdsConfig : rdsConfigs) {
-            if (rdsConfig != null) {
-                RdsView rdsView = new RdsView(rdsConfig);
-                String componentName = rdsConfig.getType().toLowerCase();
+    public TemplateModelContextBuilder withRdsConfigs(Iterable<RdsView> rdsViews) {
+        for (RdsView rdsView : rdsViews) {
+            if (rdsView != null) {
+                String componentName = rdsView.getType();
```

Comment:
Sanity check: Is it intentional that `.toLowerCase()` is missing? The original statement looked like this:
```
String componentName = rdsConfig.getType().toLowerCase();
```

Reply:
we are doing a lower when we try to get a key from this map but enyway let me add here